pkg/controllers/mcs/endpointslice_controller.go

```diff
@@ -81,7 +92,8 @@ func (c *EndpointSliceController) SetupWithManager(mgr controllerruntime.Manager
 			return util.GetLabelValue(createEvent.Object.GetLabels(), util.ServiceNameLabel) != ""
 		},
 		UpdateFunc: func(updateEvent event.UpdateEvent) bool {
-			return util.GetLabelValue(updateEvent.ObjectNew.GetLabels(), util.ServiceNameLabel) != ""
+			return (util.GetLabelValue(updateEvent.ObjectNew.GetLabels(), util.ServiceNameLabel) != "" ||
+				util.GetLabelValue(updateEvent.ObjectNew.GetLabels(), util.MultiClusterServiceNameLabel) != "")
```

Comment:
add more comments to explain why we add this

Reply:
The outermost parentheses can be omitted.